handbook/network-security/network-security.md

```diff
@@ -273,6 +273,63 @@ If you are using a self-signed certificate on your `mongod` instance you must in
 !!! note
     Ensure **Use Transport Layer Security (TLS) encryption** is enabled in the **NoSqlDatabase** section of the **SystemLink Server Configuration** application.
 
+## SSL/TLS for remote PostgreSQL Instances using Client Certificates
+
+SystemLink provides limited support connecting to PostgreSQL instances that require client certificate authentication. Refer to the [PostgreSQL documentation](https://www.postgresql.org/docs/current/ssl-tcp.html#SSL-CLIENT-CERTIFICATES) for details on how to enable TLS and how to require client certificate authentication.
+
+!!! note
+    SystemLink only supports PostgreSQL client certificates in the PFX/PKCS#12 format and does not support separate certificate and key files in the PEM or ASN.1 DER formats.
+
+### Converting PEM client key and certificate pairs to PFX
+
+After enabling client certificate authentication on your PostgreSQL instance, your database provider or administrator should provide you with a client key and a client certificate that has been signed against the PostgreSQL instance's root Certificate Authority certificate. These will often be in the PEM format. These steps will convert these PEM files into a single PFX file that SystemLink supports.
+
+1. Copy the CA certificate, any intermediate certificates, the client key, and the client certificate to your SystemLink application server.
+
+2. Open a command prompt and navigate to **"C:\Program Files\National Instruments\Shared\Web Server"**
+
+3. Run the following command, replacing the placeholders with file paths to the indicated file
+   ```bash
+   openssl.exe pkcs12 -inkey <client-key-path> -in <client-cert-path> -certfile <server-ca-path> -export -out <output-.pfx-file-path>
+   ```
+!!! note
+    If you have intermediate certificates, include them in the above command with separate `-certfile <intermediate-cert-path>` parameters
```

Comment:
does the order of the intermediate and leaf certs on the command line matter?

Reply:
No, it doesn't matter.